app_dart/lib/src/request_handlers/refresh_chromebot_status.dart

```diff
@@ -53,8 +58,30 @@ class RefreshChromebotStatus extends ApiRequestHandler<Body> {
   Future<Body> get() async {
     final LuciService luciService = luciServiceProvider(this);
     final DatastoreService datastore = datastoreProvider(config.db);
+    for (String branch in await config.flutterBranches) {
+      await _updateBranch(branch, luciService: luciService, datastore: datastore);
+    }
+    return Body.empty;
+  }
+
+  /// Update LUCI tasks in [datastore] for [branch] on the framework repo.
+  Future<void> _updateBranch(
+    String branch, {
+    LuciService luciService,
+    DatastoreService datastore,
+  }) async {
+    final List<Commit> commits = await datastore.queryRecentCommits(limit: 1, branch: branch).toList();
+    if (commits.isEmpty) {
+      log.debug('Branch $branch does not have any commits');
+      return;
+    }
+    final Commit latestCommit = commits.first;
+    final SchedulerConfig schedulerConfig = await scheduler.getSchedulerConfig(latestCommit);
+    final List<Target> postsubmitTargets = scheduler.getPostSubmitTargets(latestCommit, schedulerConfig);
+    final List<LuciBuilder> postsubmitBuilders =
+        postsubmitTargets.map((Target target) => LuciBuilder.fromTarget(target, latestCommit.slug)).toList();
     final Map<BranchLuciBuilder, Map<String, List<LuciTask>>> luciTasks = await luciService.getBranchRecentTasks(
```

Comment:
`getBranchRecentTasks` returns builds for all branches, do we need to loop through `config.flutterBranches`?

Reply:
Thanks, removed. I plan to remove this handler altogether once we schedule the prod tasks from cocoon, which will be less of a headache (we can rely what's in github checks or datastore)